### PR TITLE
Use npm as default package manager

### PR DESCRIPTION
**Pull request by meszarosrob**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Refactor

**What is the current behavior?** (You can also link to an open issue here)
Yarn is used as the default package manager.

**What is the new behavior (if this is a feature change)?**
Npm is used as the default package manager. A deprecation warning is displayed for those using the `PACKAGE_MANAGER` input.

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
There may be instances where this can cause issues since Yarn and npm are not fully interchangeable.
In such cases, the users can choose to explicitly set the `PACKAGE_MANAGER` to `yarn`.

**Other information**:
Based on the AT decision.

### CODE REVIEW

**Review by Chrico — APPROVED** [review]

File: docs/archive-creation.md
Comment (tyrann0us) [inline docs/archive-creation.md:1]: Please align double quotes in the table to single quotes according to the Boy Scout rule. Thank you!

File: docs/ddev-playwright.md
Comment (tyrann0us) [inline docs/ddev-playwright.md:1]: Same here (`"` to `'`).

**Review by tyrann0us — APPROVED** [review]
Thank you for working on this! LGTM. Left two minor comments that don't stop me from approving.

File: docs/archive-creation.md
Comment (tyrann0us) [reply docs/archive-creation.md:1]: I will handle this and more documentation cleanup in a separate PR.

**Review by tyrann0us — APPROVED** [review]
Thanks for the updates!

**Pull request merged**